stream.go

```diff
@@ -0,0 +1,126 @@
+package kinesis
+
+import (
+	"fmt"
+	"log"
+	"text/template"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kinesis"
+	"github.com/gliderlabs/logspout/router"
+)
+
+type ErrStreamNotReady struct {
+	s string
```

Comment:
Super minor, but in general it's nice to make fields like this on Error structs exported, so consumers can type assert and check individual fields:

```go
ErrStreamNotReady struct {
        Stream string
}
```